core/model/time_tracking.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
+
+import datetime
+from typing import TYPE_CHECKING
+
+from sqlalchemy import (
+    Boolean,
+    CheckConstraint,
+    Column,
+    DateTime,
+    ForeignKey,
+    Integer,
+    String,
+    UniqueConstraint,
+)
+from sqlalchemy.orm import Session, relationship
+
+from core.model import Base, get_one_or_create
+
+if TYPE_CHECKING:
+    from sqlalchemy.orm import Mapped
+
+    from core.model import Identifier
+
+
+class IdentifierPlaytimeEntry(Base):
```

Comment:
Suggestion: Since I think these entries are "identified" by more than the identifier, I think we should rename this table to `playtime_entries`.